module/VuFind/tests/integration-tests/src/VuFindTest/Auth/DatabaseTest.php

```diff
@@ -38,7 +38,7 @@
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org Main Page
  */
-class DatabaseTest extends \PHPUnit\Framework\TestCase
+final class DatabaseTest extends \VuFindTest\Integration\MinkTestCase
```

Comment:
This seems to be changing to an incorrect base class; also, why make it final?

Reply:
The incorrect class comes probably from some merge, I'll fix it

Reply:
The base class is changed because of UserCreationTrait, which calls methods `findCss`, `findCssAndSetValue` and `snooze`. These methods are defined in MinkTestCase class

Reply:
I can probably refactor to solve this. I'll try as soon as time permits.